test/benchmarks/passes.py

```diff
@@ -188,3 +194,61 @@ def setup(self, n_qubits, depth, _):
 
     def time_collect_multiq_block(self, _, __, max_block_size):
         CollectMultiQBlocks(max_block_size).run(self.dag)
+
+
+class LitinskiTransformationPassQFTBenchmarks:
```

Comment:
You could maybe dedicate this class to scaling or so -- it's more generic that "QFT" 😄